Sources/PaymentButtons/ImageAsset.swift

```diff
@@ -6,27 +6,18 @@ enum ImageAsset {
         switch button.fundingSource {
         case .payPal:
             imageAssetString = "paypal"
-
+            
```

Comment:
Nit: is this an extra space we should remove?

Reply:
Oops, this shouldn't be here. Thanks